locale/en/blog/vulnerability/september-2022-security-releases.md

```diff
@@ -7,6 +7,100 @@ layout: blog-post.hbs
 author: Vladimir de Turckheim
```

Comment:
Can't comment on the line above, but looks like the `title` metadata needs to be updated to the 23rd